azkaban-common/src/main/java/azkaban/executor/ExecutableFlowBase.java

```diff
@@ -123,6 +123,11 @@ public String getFlowId() {
     return this.flowId;
   }
 
+  // Top level flow file name
+  public String getFlowFileName(){
+    return this.flowId + ".flow";
+  }
+
```

Comment:
Is there another way to get this info? This would cover flow 2.0 syntax but not flow 1.0 (.job + .properties files). We shouldn't hardcode these extensions here anyways IMO.